sql/sql_base.cc

```diff
@@ -8329,9 +8329,9 @@ bool setup_tables(THD *thd, Name_resolution_context *context,
         (2)  Table hints are not adjusted yet
         (3)  Table is not in the INSERT part of INSERT..SELECT
       */
-      if (qb_hints &&                          // (1)
-          !table_list->opt_hints_table &&      // (2)
-          !(select_insert && is_insert_part))  // (3)
+      if (qb_hints &&                              // (1)
```

Comment:
So this condition remains unchanged, right? Then no need to change formatting or add extra brackets, let's just keep these 3 lines unmodified